x/axelarnet/exported/params.go

```diff
@@ -5,13 +5,15 @@ import (
 	tss "github.com/axelarnetwork/axelar-core/x/tss/exported"
 )
 
-var (
+const (
 	// NativeAsset is the native asset on Axelarnet
 	NativeAsset = "uaxl"
```

Comment:
why keep uaxl here?

Reply:
the value is used in a bunch of places, thoughts on how to refactor that?

Reply:
what I mean is, this PR specifically introduces a NativeAsset parameter, so why not use that?

Reply:
I thought const would be changed too via the build flag but i guess not. Switch it to a var. I do want to keep the default to `uaxl`, but it's overridden by the build flag.

Reply:
Referring this for the change: https://github.com/cosmos/cosmos-sdk/blob/main/version/version.go#L8